src/onboarding/registration/NameAndPicture.test.tsx

```diff
@@ -183,32 +192,42 @@ describe('NameAndPictureScreen', () => {
     expect(getByText('nameAndPicGuideCopyContent')).toBeTruthy()
   })
   it('does not render skip button when configured so', () => {
-    // TODO replace route param with mock Statsig flag
+    ExperimentParams[StatsigLayers.NAME_AND_PICTURE_SCREEN].showSkipButton.defaultValue = false
+    Statsig.getLayer = jest
+      .fn()
+      .mockImplementation(() => ({ get: jest.fn().mockReturnValue(false) }))
```

Comment:
instead of overriding this again here, you can do something like
```js
const mockStatsigGet = jest.fn().mockReturnValue(null) // default implementation
jest.mock('statsig-react-native', () => ({
  Statsig: {
    getLayer: jest.fn().mockReturnValue({get: mockStatsigGet})
  },
}))
```
and then inside specific functions where you want a different response, you can do
```js
mockStatsigGet.mockReturnValueOnce(false)  // or true
```